### PR TITLE
v1.4.0

### PR DESCRIPTION
**Pull request by TheKeymaster**

### CODE REVIEW

**Comment by TheKeymaster** [comment]
@georgms I have done a few things. First up:

 * If this PR is merged we do have :boom:  :boom:  :boom:  **100% Code Coverage** :boom:  :boom:  :boom: 
 * Tests take longer because some tests were not running properly before.
 * Missing tests were added to get this Code Coverage.

TODO NOW:

 * Automated test management.

**Comment by TheKeymaster** [comment]
@georgms could you provide me a `GITHUB OAUTH TOKEN` for this repository?  
It must have the `public_repo` or `repo` scope to upload assets and please make sure it is encrypted ;).

File: _test/template.response.test.php [hunk 1]
```diff
@@ -314,6 +321,18 @@ public function test_response_has_notify_message_when_five_hundred_pages_have_no
...
+    public function test_plugin_is_sorted_at_the_top() {
+        $adminPlugin = new admin_plugin_findologicxmlexport();
+        $expectedSortValue = 1;
+        $this->assertEquals($expectedSortValue, $adminPlugin->getMenuSort());
+    }
+
+    public function test_plugin_not_requires_to_be_superuser() {
+        $adminPlugin = new admin_plugin_findologicxmlexport();
+        $adminCanAccessAswell = true;
+        $this->assertEquals($adminCanAccessAswell, $adminPlugin->forAdminOnly());
```
Comment (georgms) [inline _test/template.response.test.php:333]: `assertTrue`

**Comment by georgms** [comment]
@TheKeymaster Please provide info on how to generate the OAuth token.

**Review by georgms — CHANGES_REQUESTED** [review]

**Comment by TheKeymaster** [comment]
@georgms OAuth tokens can be generated in [Personal Access Tokens](https://github.com/settings/tokens). Please send me this token via mail, because it would require much things to install etc. in order to encrypt the API Key. I already got those things installed, so that's the easier way.

**Comment by georgms** [comment]
@TheKeymaster I don't feel comfortable giving you my **personal** access token. Are you sure this is the right way to go?

**Comment by TheKeymaster** [comment]
@georgms you don't need to. You can encrypt your key via the Travis command line tool (which is a ruby gem) by yourself. Please follow these steps if you want to do so:

### If you do not have ruby

#### Installing ruby version manager (rvm)

```bash
> sudo apt-get --assume-yes update  
> command curl -sSL https://rvm.io/mpapis.asc | gpg --import -  
> curl -L https://get.rvm.io | bash -s stable  
> source ~/.rvm/scripts/rvm  
```

#### Installing ruby with rvm

```bash
> rvm install 2.3.0  
> rvm use 2.3.0 --default  
> ruby -v  
```

`ruby -v` to check if the installed version matches.

### If you do not have ruby-dev or gem

#### Installing ruby-dev and gem

```bash
> sudo apt-get install ruby2.3-dev
> sudo apt-get install gem
```

### Installing travis gem

```bash
> gem install travis
```

If you can't execute `travis`, then you might need to reboot your computer or re-login again.

### Encrypt your OAuth key

See [Encryption keys](https://docs.travis-ci.com/user/encryption-keys/).

**Comment by TheKeymaster** [comment]
See #44

**Pull request closed without merging**